src/mixins/filing-template-mixin.ts

```diff
@@ -105,7 +105,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // if filing...
     if (!isDraft) {
       // Filter out removed parties and delete "actions" property
-      parties = parties.filter(x => !x.actions.includes(ActionTypes.REMOVED))
+      parties = parties.filter(x => !x.actions?.includes(ActionTypes.REMOVED))
```

Comment:
This fixes the error when a party has no actions (eg, not added/edited/removed).